added planet_detail.html file along with an update to urls.py and views.py

mdhamid1 opened the pull request:
Added a planet_detail view which included creating a new template file called planet_detail.html. Added a new URL pattern in the urls.py file that dynamically looks at the planet primary key and adds "planet_detail" at the end of it such that when someones goes to BASE_DIR/planets/[planet_id]/planet_detail it will take them to a page that contains a summary of the planet details. Then updated the views.py file to include a detail view for the planet instance.

arwhyte commented on apps/webapp/templates/webapp/planet_detail.html:
```diff
@@ -0,0 +1,120 @@
+                <p>{{ planet_detail.name }}</p>
+                </div>
+            </div>
+
```
Remove blank line between `</div>` and end if tag throughout this file.

arwhyte commented on apps/webapp/views.py:
```diff
@@ -101,3 +101,11 @@ class VehicleListView(generic.ListView):
+
+class PlanetDetailView(generic.DetailView):
+	model = Planet
+	context_object_name = 'planet_detail'
```
Change string value assigned to `context_object_name` to `planet` (that's the object you are working with; the trailing `_detail` is a view/template-related name.  Then update references in `planet_detail.html`.

arwhyte commented on apps/webapp/templates/webapp/planet_detail.html:
```diff
@@ -0,0 +1,120 @@
+        <h2>Planet Details</h2>
+        </header>
+
+        {% if planet_detail.name %}
```
change the object name throughout the file from `planet_detail` to `planet`.

arwhyte reviewed (changes requested):
See comments.  object name change and minor formatting fix.  Make changes, add, commit, and push to your repo and they will be automatically added to this open PR.

mdhamid1 commented:
please note that in addition to the requested changes from the pull request review, I also made a very small update to the urls.py file to include a "/" at the end of the url for planet_detail. 
-old url path: 'planets/<int:pk>/planet_detail'
-new url path: 'planets/<int:pk>/planet_detail/'

mdhamid1 commented:
Made suggested changes to planet detail view implementation as per discussion in team meeting on 02/05. Changes are shown below: 
-removed "planet_detail" from end of url for PlanetDetailView in urls.py
-changed context_object_name variable from "planet" to "planets" in views.py 
-removed get_queryset() method and added get_object() method PlanetDetailView class in views.py
-changed all occurrences of planet to planets in planet_detail.html
-got rid of the space between the article tag and the last endif statement in planet_detail.html
-added href link in planets.html to allow user to navigate to the page containing planet details
